bootstrap_salt/ec2.py

```diff
@@ -52,38 +53,41 @@ def get_instance_by_id(self, inst_id):
         resv = self.conn_ec2.get_all_reservations([inst_id])
         return [i for r in resv for i in r.instances][0] if resv else None
 
-    def get_master_instance(self, stack_name_or_id, master_tag_name='SaltMaster'):
-        stack_instances = self.cfn.get_stack_instances(stack_name_or_id)
-        stack_instance_ids = [x.instance_id for x in stack_instances]
+    def get_master_instance(self, master_tag_name='SaltMaster'):
         filters = {'tag-key': master_tag_name,
-                   'instance-state-name': 'running',
-                   'instance-id': stack_instance_ids}
```

Comment:
Yep - I did this for two reasons:

1) Currently there's no functionality in the bootstrap cfn stuff to create a salt master. So I had to manually create it outside of the cloudformation stack. (I've since found a way to do it inside the stack by adding a ```includes:``` entry into the JSON and some manual yaml - thanks Matthew).

2) I thought we were using sub-accounts to try and separate things? So we'd be using a separate AWS login for the different account types? This effectively means we'd have one stack per aws account.

I'm aware this is a breaking change :)